shared_python/Tags.py

```diff
@@ -47,7 +47,9 @@ def create_tags_table(self, database = None):
         `ao3_tag` varchar(1024) DEFAULT NULL,
         `ao3_tag_type` VARCHAR(255) DEFAULT NULL,
         `ao3_tag_category` VARCHAR(255) DEFAULT NULL,
-        `ao3_tag_fandom` VARCHAR(255) DEFAULT NULL
+        `ao3_tag_fandom` VARCHAR(255) DEFAULT NULL,
+        PRIMARY KEY (`id`),
+        UNIQUE KEY `id_UNIQUE` (`id`)
```

Comment:
You don't need an additional unique key here; primary keys are always unique, so this will just create a duplicate index.

Reply:
Ah good catch! All the other table definitions have this too because I probably adapted some table definition that needed uniqueness on another field, and didn't question whether we needed that on the primary key. However, I decided not to scope creep this PR; I've opened OD-581 for someone to do later.